docs/install/index.rst

```diff
@@ -0,0 +1,38 @@
+.. _install:
+
+Install Kolibri
+###############
+
+See the `Kolibri changelog here <https://learningequality.org/r/kolibri-changes>`_ to review specific changes in each version of Kolibri. Documentation for `previous Kolibri versions are available here <https://readthedocs.org/projects/kolibri/>`_.
+
+
+Learn about whether Kolibri will run on your system, and how to get the server set up:
+
+.. toctree::
+  :maxdepth: 1
+
+  system_requirements
+  initial_setup
```

Comment:
This step (our _setup wizard_) logically goes **after** the installation, so I don't think this is the best place.